src/patternfly/components/Button/button.scss

```diff
@@ -79,26 +77,25 @@
 
   // Link btn
   --pf-c-button--m-link--BackgroundColor: transparent;
-  --pf-c-button--m-link--PaddingRight: var(--pf-global--spacer--xs);
-  --pf-c-button--m-link--PaddingLeft: var(--pf-global--spacer--xs);
   --pf-c-button--m-link--Color: var(--pf-global--link--Color);
   --pf-c-button--m-link--hover--Color: var(--pf-global--link--Color--hover);
   --pf-c-button--m-link--hover--BackgroundColor: transparent;
```

Comment:
can we remove the transparent background for default and hover state for both the link and the plain?

Reply:
yep! good catch